tox.ini

```
@@ -1,10 +1,12 @@
 [tox]
 isolated_build = true
-envlist = py39, flake8, mypy, black
+envlist = py38, py39, py310, flake8, mypy, black
 
 [gh-actions]
 python =
+    3.8: py38, flake8, mypy, black
     3.9: py39, flake8, mypy, black
+    3.10: py310, flake8, mypy, black
```

The static code analysis shouldn't be python version dependent. Let's just run it for the latest python.
We only need to run pytest for each python version:
```suggestion
    3.8: py38
    3.9: py39
    3.10: py310, flake8, mypy, black
```

---

right, I remembered you said something about this but didn't remember what exactly, so this is it. 🆒